htdocs/js/jquery.dynamictable.js

```diff
@@ -98,8 +164,25 @@
                 checkOverflow(table.parentElement, rightLink, leftLink);
             });
 
+            if (options && options.freezeColumn) {
+                column = $("#" + options.freezeColumn);
+                columnNumber = $(column).parent().children().index($(column));
+                $(this).find("tr").each(function (key, value) {
+                    if (key === 0) {
+                        var child2 = $(value).children().get(columnNumber + 1);
+                        $(child2).attr('class', id + 'Next');
+                    }
+                    child1 = $(value).children().get(columnNumber);
+                        // height = $(child1).next().outerHeight();
+                    $(child1).attr('class', id + "FrozenColumn");
```

Comment:
This should use $(child1).addClass instead of $(child1).attr('class', ...), because otherwise this line will eliminate other classes that might have been added to the element (I think).